tests/test_compute_ho_ver_maps.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.transforms.intensity.array import ComputeHoVerMaps
+from monai.utils import min_version, optional_import
+from tests.utils import TEST_NDARRAYS, assert_allclose
+
+skimage, has_skimage = optional_import("skimage", "0.19.0", min_version)
+
+
+INSTANCE_MASK = np.zeros((16, 16), dtype="int16")
```

Comment:
please use https://github.com/Project-MONAI/MONAI/blob/1941283b1862e3f578ccfeb2bdfa864171125029/monai/data/synthetic.py#L21 to write more compact use cases.

Reply:
That's good idea but here I need to manually calculate the normalized distances to the center of mass of each region, so I cannot work with a randomly generated mask. Does it make sense?